src/pages/ListTableList/_mock.ts

```diff
@@ -0,0 +1,146 @@
+import { Request, Response } from 'express';
+import { parse } from 'url';
+import { TableListItem, TableListParams } from './data.d';
+
+// mock tableListDataSource
+const genList = (current: number, pageSize: number) => {
+  const tableListDataSource: TableListItem[] = [];
+
+  for (let i = 0; i < pageSize; i += 1) {
+    const index = (current - 1) * 10 + i;
+    tableListDataSource.push({
+      key: index,
+      disabled: i % 6 === 0,
+      href: 'https://ant.design',
+      avatar: [
+        'https://gw.alipayobjects.com/zos/rmsportal/eeHMaZBwmTvLdIwMfBpg.png',
+        'https://gw.alipayobjects.com/zos/rmsportal/udxAbMEhpwthVVcjLXik.png',
+      ][i % 2],
+      name: `TradeCode ${index}`,
+      title: `一个任务名称 ${index}`,
+      owner: '曲丽丽',
+      desc: '这是一段描述',
+      callNo: Math.floor(Math.random() * 1000),
+      status: Math.floor(Math.random() * 10) % 4,
+      updatedAt: new Date(`2017-07-${Math.floor(i / 2) + 1}`),
+      createdAt: new Date(`2017-07-${Math.floor(i / 2) + 1}`),
+      progress: Math.ceil(Math.random() * 100),
+    });
+  }
+  return tableListDataSource;
+};
+
+let tableListDataSource = genList(1, 100);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 100](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4665582)